packages/shared/components/FileTransfer/FileTransferStateless/FileList/FileListItem.tsx

```diff
@@ -102,7 +102,7 @@ const ProgressPercentage = styled(Text)`
   width: 36px;
 `;
 
-const Li = styled.li`
+const Li = styled('li')`
```

Comment:
`styled.li`?

Reply:
Originally the styled implementation didn't work with `styled.element`, but I had fixed that, so I'll revert this